packages/grant-explorer/src/features/round/ViewCartPage/SummaryContainer.tsx

```diff
@@ -43,6 +47,71 @@ export function SummaryContainer() {
     [projects]
   );
 
+  const chainIds = useMemo(
+    () => Object.keys(projectsByChain).map(Number),
+    [projectsByChain]
+  );
```

Comment:
Unless the processed data is big, the performance increase in using `useMemo` doesn't outweigh the decrease in readability.